desktop/cmp/grid/ColChooserModel.js

```diff
@@ -52,9 +52,9 @@ export class ColChooserModel {
 
```

Comment:
Sorry should have called this out and given some context. When testing the action col, I found that it wasn't working with the column chooser and found it was because the action col does not specify a field (nor should it), which led to this change. I would consider this a bug, as the field is an optional property on the column and is not guaranteed to be unique, we shouldn't be using it as a unique identifier for columns.

Reply:
great! Nice fix